GiniVision/Classes/Core/FilePickerManager.swift

```diff
@@ -24,18 +26,21 @@ internal final class FilePickerManager: NSObject {
         }
     }
     
+    override init() {
+        super.init()
+        DispatchQueue.global().async {
+            self.galleryCoordinator.start()
```

Comment:
I don't like having an async operation in a constructor. A constructor would not have side effects.
And I fear, that's not what people would expect from a class.

Reply:
What about this one?